cves/CVE-2011-3944.yml

```diff
@@ -306,8 +306,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: All files should be scanned for common issues before read by the system.
```

Comment:
This is also a `complex_inputs` issue, because smack files are complex inputs.

Reply:
Not all the time. They are just regular files to the average user. To us, they are complex, but for the average user, they are not.